org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -559,12 +559,32 @@
             id="org.eclipse.wildwebdeveloper.eslint"
             label="ESLint"
             clientImpl="org.eclipse.wildwebdeveloper.eslint.ESLintClientImpl"/>
+      <!-- See https://github.com/microsoft/vscode-eslint/blob/main/server/src/languageDefaults.ts#L14
+      about language id -->
       <contentTypeMapping
             contentType="org.eclipse.wildwebdeveloper.js"
-            id="org.eclipse.wildwebdeveloper.eslint"/>
+            id="org.eclipse.wildwebdeveloper.eslint"
+            languageId="javascript"/>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.ts"
+            id="org.eclipse.wildwebdeveloper.eslint"
+            languageId="typescript"/>
       <contentTypeMapping
             contentType="org.eclipse.wildwebdeveloper.ts"
-            id="org.eclipse.wildwebdeveloper.eslint"/>
+            id="org.eclipse.wildwebdeveloper.html"
+            languageId="html"/>
+      <contentTypeMapping
+            contentType="org.eclipse.wildwebdeveloper.jsx"
+            id="org.eclipse.wildwebdeveloper.eslint"
+            languageId="javascriptreact"/>
+      <contentTypeMapping
```

Comment:
It is important that you keep javascript as langage id because otherwise inlay hint  préférences will not work.

Indeed typescript langage server uses this langage id to apply proper preferences

Reply:
That's not how the eslint langauge server works.

Reply:
Thats annoying. We will l'oose i.lay hint préférences for jsx and tsx